src/components/SelectInput/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+
+import { BsCaretDownFill } from "react-icons/bs"
+import Select, {
+  components,
+  DropdownIndicatorProps,
+  PlaceholderProps
+} from "react-select"
+
+import { SelectStyled } from "./styles"
+import { Props } from "./types"
+
+export const SelectInput = ({
+  options,
+  isMulti,
+  placeholder = "",
+  ...props
+}: Props) => {
+  const optionsValues = options.map((option) => ({
+    value: option,
+    label: option
```

Comment:
Vamos passar as options como um objeto para ter mais flexibilidade nas labels e valores.

```suggestion
    value: option.value,
    label: option.label
```